excitebike/src/test/java/com/codenjoy/dojo/excitebike/model/PlayersSpawnSystemParametrizedTest.java

```diff
@@ -66,8 +64,8 @@ public PlayersSpawnSystemParametrizedTest(String name, int newPlayerNumberAfterI
     @Parameterized.Parameters(name = "{0}")
     public static Collection<Object> data() {
         return Lists.newArrayList(
-                new Object[]{"shouldAddThreeBikesInFirstChessColumn",
-                        0,
+                /*new Object[]{"1. shouldAddThreeBikesToFirstColumn",
```

Comment:
do we need this commented test?

Reply:
Yes, we do! I forgot to un-comment some test cases after debugging. 
Done now.
Thank you for the notice!